keel-orca/src/main/kotlin/com/netflix/spinnaker/keel/orca/ClusterExportHelper.kt

```diff
@@ -0,0 +1,106 @@
+package com.netflix.spinnaker.keel.orca
+
+import com.netflix.spinnaker.keel.clouddriver.CloudDriverService
+import com.netflix.spinnaker.keel.core.api.ClusterDeployStrategy
+import kotlinx.coroutines.async
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+
+/**
+ * Provides common logic for multiple cloud plugins to export aspects of compute clusters.
+ */
+@Component
+class ClusterExportHelper(
```

Comment:
I wasn't sure in which module to put this class since it has a dependency on `keel-orca` and `keel-clouddriver`. It turned out that `keel-orca` already had a dependency on `keel-clouddriver`, so I decided to put it here. 🤷‍♂️

Reply:
Sounds reasonable to me!